public/app/features/alerting/unified/hooks/usePanelCombinedRules.ts

```diff
@@ -20,70 +10,19 @@ interface Options {
 }
 
 interface ReturnBag {
-  errors: SerializedError[];
+  errors: unknown[];
   rules: CombinedRule[];
 
   loading?: boolean;
 }
 
 export function usePanelCombinedRules({ dashboardUID, panelId, poll = false }: Options): ReturnBag {
-  const dispatch = useDispatch();
-
-  const promRuleRequest =
-    useUnifiedAlertingSelector((state) => state.promRules[GRAFANA_RULES_SOURCE_NAME]) ?? initialAsyncRequestState;
-  const rulerRuleRequest =
-    useUnifiedAlertingSelector((state) => state.rulerRules[GRAFANA_RULES_SOURCE_NAME]) ?? initialAsyncRequestState;
-
-  // fetch rules, then poll every RULE_LIST_POLL_INTERVAL_MS
-  useEffect(() => {
-    const fetch = () => {
-      dispatch(
-        fetchPromRulesAction({
-          rulesSourceName: GRAFANA_RULES_SOURCE_NAME,
-          filter: { dashboardUID, panelId },
-        })
-      );
-      dispatch(
-        fetchRulerRulesAction({
-          rulesSourceName: GRAFANA_RULES_SOURCE_NAME,
-          filter: { dashboardUID, panelId },
-        })
-      );
-    };
-    fetch();
-    if (poll) {
```

Comment:
🤔 I am not familiar with this part, but in the new hook, it seems we are not implementing the `polling` we have here. From what I can see, it looks like we were using this `poll` boolean to re-fetch data in regular intervals.

Reply:
thanks @axelavargas ! this is a very good catch!!! I will add the polling interval to the queries in the new  `useCombinedRules` hook